frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeInputOutputTab.tsx

```diff
@@ -88,12 +101,26 @@ const SelectedNodeInputOutputTab: React.FC<SelectedNodeInputOutputTabProps> = ({
     [getExecutionValueFromInputType],
   );
 
-  if (!task.inputs && !task.outputs) {
+  if (!task.inputs && !task.outputs && !execution) {
     return <>No content</>;
   }
 
   return (
     <Stack hasGutter>
+      {execution?.id && (
+        <StackItem data-testid="execution-name">
+          <Grid hasGutter>
+            <GridItem span={6}>
+              <b>Execution name</b>
+            </GridItem>
+            <GridItem span={6}>
+              <Link to={executionDetailsRoute(namespace, execution.id.toString())}>
+                {getExecutionDisplayNameFromObject()}
+              </Link>
+            </GridItem>
+          </Grid>
+        </StackItem>
+      )}
```

Comment:
we should hide this if the experiments feature flag is disabled (meaning we are hiding the experiments feature)

For reference, `disablePipelineExperiments` feature flag is being used to keep all the artifact execution, experiment, and those related details out of the release by hiding it behind the feature flag. In 2.11 we will remove this flag